### PR TITLE
Add a blog post for the 1.6 release

### PR DESCRIPTION
**Pull request by annajung**
Signed-off-by: Anna Jung (VMware) <antheaj@vmware.com>

- Add a blog post for the 1.6 release

cc @kubeflow/release-team @jbottum

### CODE REVIEW

**Comment by jbottum** [comment]
@johnugeorge do the Training Operator Python SDK and Clientset support tensorflow too ?

**Comment by jbottum** [comment]
@kimwnasptd could you please review the Notebook and Manifest section ?

**Comment by annajung** [comment]
deploy preview: https://deploy-preview-124--infallible-murdock-d05108.netlify.app/kubeflow-1.6-release/

**Comment by jbottum** [comment]
@pvaneck would you please review the KServe info in this post?

**Comment by jbottum** [comment]
@james-jwu @zijianjoy would you guys please review the KFP content and make sure the links are correct ?

**Comment by jbottum** [comment]
> @johnugeorge do the Training Operator Python SDK and Clientset support tensorflow too ?

per our conversation on 8/25, tensorflow is supported in both and the other frameworks are the enhancement.

File: _posts/2022-08-24-kubeflow-1.6-release.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+<br />
+
+KServe has these breaking changes to the Python SDK:
+
+- KFModel is renamed to Model
+- KFServer is renamed to ModelServer
+- KFModelRepository is renamed to ModelRepository
+
+To learn more please see the [KServe v0.8 release notes](https://github.com/kserve/kserve/releases/tag/v0.8.0) and
+ Release [blog post](https://kserve.github.io/website/0.9/blog/articles/2022-02-18-KServe-0.8-release/)
```
Comment (shivaylamba) [inline _posts/2022-08-24-kubeflow-1.6-release.md:128]: Adding a full stop at end of the line

File: _posts/2022-08-24-kubeflow-1.6-release.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+ applied, the Pods of components that can become ready, and the basic objects that can be created.
+ 
+In parallel to developing these new software features, the Kubeflow Community also completed its annual user survey.
+ The survey generated many good user insights into requirements for Kubeflow, and you can see more on the
+ [survey results](https://docs.google.com/document/d/1TLl8iFWQ-x2pVAHV9G8GFYMgAOuN2VFOnZV2JQNA-uc/edit#). Of the many
+ highlights, we would like to identify the growing request for model monitoring as shown in the chart below.
+
+![2022 User Survey Gaps In ML Activities and Workflow](/images/2022-08-24-kubeflow-1.6-release/2022-user-survey-gaps-in-ml-activities-and-workflow.png)
+
+We believe the increased focus on monitoring identifies Kubeflow’s maturing user base. The Community is making efforts
```
Comment (shivaylamba) [inline _posts/2022-08-24-kubeflow-1.6-release.md:32]: small case for community

File: _posts/2022-08-24-kubeflow-1.6-release.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+<br />
+
+For model serving, Kubeflow v1.6 incorporates the KServe v0.8.0 release, which includes these enhancements:
+
+- ClusterServingRuntime and ServingRuntime CRDs 
+- A new Model Spec is introduced to the InferenceService Predictor Spec as a new way to specify models.
+- Support for Knative 1.0+ 
+- gRPC for transformer to predictor network communication
+- An alternative backend ModelMesh with multi-namespace support
```
Comment (pvaneck) [inline _posts/2022-08-24-kubeflow-1.6-release.md:117]: Suggest having this list item be:

`Multi-namespace support for the ModelMesh alternative backend`

File: _posts/2022-08-24-kubeflow-1.6-release.md (hunk 1 above)
Comment (annajung) [reply _posts/2022-08-24-kubeflow-1.6-release.md]: Added . at the end

File: _posts/2022-08-24-kubeflow-1.6-release.md (hunk 2 above)
Comment (annajung) [reply _posts/2022-08-24-kubeflow-1.6-release.md:32]: done

File: _posts/2022-08-24-kubeflow-1.6-release.md (hunk 3 above)
Comment (annajung) [reply _posts/2022-08-24-kubeflow-1.6-release.md]: Updated

File: _posts/2022-08-24-kubeflow-1.6-release.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+ provides a good overview of v2. 
+
+<br />
+
+The KFP v2 alpha introduces: 
+
+- An Argo-agnostic approach for creating and running pipelines 
+- A brand new DAG visualization, which uses the Pipeline Template and MLMD in this [pipeline spec](https://github.com/kubeflow/pipelines/blob/master/api/v2alpha1/pipeline_spec.proto)
+- Ability to set pipeline root for the root path of object storage when building pipeline template, and when creating pipeline run.
+- PIP install functionality from private PyPI repository [design](https://docs.google.com/document/d/1r7XKNncJ6du1EnX3iiUqOJXMMA2nvoXXUarOXe_sm1o/edit?resourcekey=0-pGC-6oMc2Iog5hKW1eJHAA#heading=h.x9snb54sjlu9)
```
Comment (jbottum) [inline _posts/2022-08-24-kubeflow-1.6-release.md:53]: The pipelines WG says that existing lines 52 and line 53 should be deleted and replace with this line...

[Streamlined Component Authoring] (https://docs.google.com/presentation/d/1HzMwtI2QN67xQp2lSxmuXhitEsukLB7mvZx4KAPub3A/edit#slide=id.gecbd775238_0_438)

Comment (annajung) [reply _posts/2022-08-24-kubeflow-1.6-release.md]: Done

File: _posts/2022-08-24-kubeflow-1.6-release.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+For those users who are still relying on the KFP v1 functionality, Kubeflow v1.6 and its KFP v2 component are fully
+ tested and supported with the mature features in the legacy Kubeflow Pipelines v1.8 SDK. This provides the same
+ functionality that thousands of KFP v1 users leverage in production today.   
+
+<br />
+
+In addition to the KFP v2 alpha, Kubeflow 1.6 includes feature enhancements and operational improvements for Katib
+ users. These include support for:
+
+- The population based training algorithm, URL
```
Comment (johnugeorge) [inline _posts/2022-08-24-kubeflow-1.6-release.md:73]: URL ?

Comment (annajung) [reply _posts/2022-08-24-kubeflow-1.6-release.md]: In the original google docs, it looks like the word "URL" was added to propose adding a link to the paper here. I removed the word "URL" and if you want to provide a paper that you want to link, I can link it here

**Comment by jbottum** [comment]
Some proposed changes.  @annajung I am still having trouble making edits inline, not sure why but maybe you can review these proposed changes.

Line 2 (to use alpha instead of experimental)
Kubeflow v1.6 delivers support for Kubernetes v1.22 and introduces an alpha release of Kubeflow Pipeline v2 functionality

Line 13-14 (added words to make more readable)
The Kubeflow Community is excited to announce the availability of the Kubeflow v1.6 software release, which includes support for Kubernetes v1.22 and introduces an alpha release of Kubeflow Pipeline v2 (KFP v2) functionality.

Line 19-21 (note - this includes deleting some current content and replacing with below)
Additionally, 1.6 cleans up a few CVEs in the central dashboard and enables PodDefaults webhook to pick-up new certificate updates. For the Kubernetes upgrade, the community developed and tested each Kubeflow component manifest for Kubernetes v1.22. As v1.22 introduced some breaking changes, the upgrade was a team effort, and this Kubeflow release management process will be useful in the community's future updates of Kubernetes and other software dependencies.

Line 35 (just adding v to 1.6)
To round back to the current software delivery, the following takes a deeper look at the v1.6 highlights in more detail.

Line 41 (change name of slide deck to match the name of the slide deck)
change from "Understand KFP" to "KFP v2 Introduction"

File: _posts/2022-08-24-kubeflow-1.6-release.md [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+ the user survey results and Kubeflow Working Group roadmaps will be discussed in the upcoming Kubeflow Summit. You can
+ learn more about the Kubeflow Summit [here](https://docs.google.com/document/d/1fLg1EqVtJekcXxT8GM_Rqg0-G-vIDVCrzuxE06Oq9dI/edit).
+
+To round back to the current software delivery, the following takes a deeper look at the v1.6 highlights in more detail.
+
+## Kubeflow v1.6 details
+
+From a new feature introduction standpoint, Kubeflow v1.6 includes an experimental release of the KFP v2’s new front-end,
+ back-end and SDK, which deliver a modern UI and DAG, first class support for metadata, and a simplified component
+ authoring experience. This slide deck, [KFP v2 Introduction](https://docs.google.com/presentation/d/1HzMwtI2QN67xQp2lSxmuXhitEsukLB7mvZx4KAPub3A/edit#slide=id.gecbd775238_0_20),
```
Comment (jbottum) [inline _posts/2022-08-24-kubeflow-1.6-release.md:41]: changed name of slide deck to match the actual name of the presentation (KFP v2 Introduction)

File: _posts/2022-08-24-kubeflow-1.6-release.md [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+ highlights, we would like to identify the growing request for model monitoring as shown in the chart below.
+
+![2022 User Survey Gaps In ML Activities and Workflow](/images/2022-08-24-kubeflow-1.6-release/2022-user-survey-gaps-in-ml-activities-and-workflow.png)
+
+We believe the increased focus on monitoring identifies Kubeflow’s maturing user base. The Community is making efforts
+ to explain Kubeflow’s current functionality and build into the users’ new model monitoring requirements. In addition,
+ the user survey results and Kubeflow Working Group roadmaps will be discussed in the upcoming Kubeflow Summit. You can
+ learn more about the Kubeflow Summit [here](https://docs.google.com/document/d/1fLg1EqVtJekcXxT8GM_Rqg0-G-vIDVCrzuxE06Oq9dI/edit).
+
+To round back to the current software delivery, the following takes a deeper look at the v1.6 highlights in more detail.
```
Comment (jbottum) [inline _posts/2022-08-24-kubeflow-1.6-release.md:35]: add v to v1.6

**Review by jbottum — CHANGES_REQUESTED** [review]
proposed changes in several places

**Comment by jbottum** [comment]
Future To Do: We should consider adding a link in the blog post to the Kubeflow 1.6 video presentation.   We do not have that link, which will be on Youtube.    Once that link is up, we should consider adding it to the blog post, perhaps at the end of the summary paragraph near the top to the post i.e. ~ The Kubeflow Working Group leads and Release Team have recorded a review of Kubeflow v1.6's features, which you can view here (link TBD).

**Comment by jbottum** [comment]
@johnugeorge @zijianjoy @kimwnasptd @annajung I am trying to finish this up for posting, target Thursday.   Could you please provide your /lgtm ?   thanks.

**Comment by annajung** [comment]
lgtm after Josh's changes

**Review by shivaylamba — APPROVED** [review]

**Review by charlesa101 — APPROVED** [review]

**Review by terrytangyuan — APPROVED** [review]
LGTM. Thanks everyone from the release team!

File: _posts/2022-08-24-kubeflow-1.6-release.md [hunk 8]
```diff
@@ -0,0 +1,143 @@
...
+
+- An Argo-agnostic approach for creating and running pipelines 
+- A brand new DAG visualization, which uses the Pipeline Template and MLMD in this [pipeline spec](https://github.com/kubeflow/pipelines/blob/master/api/v2alpha1/pipeline_spec.proto)
+- [Streamlined Component Authoring](https://docs.google.com/presentation/d/1HzMwtI2QN67xQp2lSxmuXhitEsukLB7mvZx4KAPub3A/edit#slide=id.gecbd775238_0_438)
+
+<br />
+
+You can learn more about KFP v2 and the related breaking changes in these docs:
+
+- [KFP SDK V2 CLI design](ahttps://docs.google.com/document/d/1Z79ebKaf3hjyadSRsme1Qo7F-Tqu4qM1cyKuLeu2PgA/edit)
```
Comment (chensun) [inline _posts/2022-08-24-kubeflow-1.6-release.md:56]: I think the slide "Understanding KFP v2" [1] is still a good overall reference for learning more about KFP v2.
In comparison, "KFP SDK V2 CLI design" is not essential to KFP v2, and can be skipped in this context. 

[1] https://docs.google.com/presentation/d/1HzMwtI2QN67xQp2lSxmuXhitEsukLB7mvZx4KAPub3A/edit#slide=id.gb4a3fac3a8_7_1911

Other than this, LGTM. Thanks!

Comment (annajung) [reply _posts/2022-08-24-kubeflow-1.6-release.md]: Replaced the KFP SDK v2 CLI design link with the KFP v2 introduction link

**Comment by chensun** [comment]
/lgtm

Thanks!

@james-jwu for approval.

File: _posts/2022-08-24-kubeflow-1.6-release.md (hunk 8 above)
Comment (jbottum) [reply _posts/2022-08-24-kubeflow-1.6-release.md]: @annajung lines 41 and lines 56 both refer to users to the KFP v2 Introduction slide deck.   We might delete line 56.

**Pull request merged**